profiles/BoseSj.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/58129377/211534605-44b02824-5f70-43eb-bd3f-92ca72c85a0f.png",
```

Comment:
I suggest more of a image like this:
![image](https://user-images.githubusercontent.com/91655303/211567811-5f474ba2-a1e5-41e1-b9f0-fce7f42fd88e.png)


```suggestion
  "image": "https://user-images.githubusercontent.com/91655303/211567625-1e497baa-9b02-4376-a688-ac000ac894d5.png",
```

Reply:
Okay i see, you mean the animated "Contribution Graph" should be visible like that? Right?

Reply:
Okay I made the changes, please review it.